### PR TITLE
Add idols info

### PR DESCRIPTION
**Pull request by Xwilarg**
- Add JSON thingy in members.js to add the different Hololive idol with their info
- Display a colored square on the calendar on their debut date (color depend of the generation)
- Add meta properties in index.html

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -62,4 +62,12 @@ body {
     margin-bottom: 5px;
     font-size: 12px;
     color: #777;
+}
+
+.EN {
```
Comment (AmayaHena) [inline css/index.css:70]: Add some comment, since css is a not that explicit

File: css/index.css [hunk 2]
```diff
@@ -62,4 +62,12 @@ body {
...
+    border: 5px solid red;
+}
+
+.JP {
```
Comment (AmayaHena) [inline css/index.css:75]: Add some comment, since css is a not that explicit

File: index.html [hunk 3]
```diff
@@ -2,7 +2,10 @@
 <html>
     <head>
```
Comment (AmayaHena) [inline index.html:3]: Some indentation to make the file prettier, and easy to read

**Review by AmayaHena — CHANGES_REQUESTED** [review]
Minor changes

File: css/index.css (hunk 1 above)
Comment (Xwilarg) [reply css/index.css:70]: Fixed with 87238d36a2b5be6dcda0c0570ff29ecb46a342f7

File: css/index.css (hunk 2 above)
Comment (Xwilarg) [reply css/index.css:75]: Fixed with 87238d36a2b5be6dcda0c0570ff29ecb46a342f7

File: index.html (hunk 3 above)
Comment (Xwilarg) [reply index.html:3]: Fixed with 8ff662616f76f10c7d0c72a94a31d1216c4439f8

**Review by AmayaHena — APPROVED** [review]

**Pull request merged**